specs/001-kiro-specs-knowledge/tasks.md

```diff
@@ -0,0 +1,174 @@
+# Tasks: Knowledge Graph Systems for Agentic Neurodata Conversion
```

Comment:
hmmm... a scary number of test for a single spec components. we have quite a few spec components to implement/coordinate. perhaps agents can work faster with clearly specified  tasks though.